install/tools/ipactl

```diff
@@ -140,10 +141,16 @@ def version_check():
     except (installutils.UpgradeMissingVersionError,
             installutils.UpgradeDataOlderVersionError) as exc:
         emit_err("IPA version error: %s" % exc)
-        emit_err("Upgrade required: please run ipa-server-upgrade command")
-        raise IpactlError("Aborting ipactl")
     except installutils.UpgradeVersionError as e:
         emit_err("IPA version error: %s" % e)
+    else:
+        return
+
+    emit_err("Automatically running upgrade")
+    try:
+        ServerUpgrade.run_cli()
```

Comment:
Like this, for example.

Reply:
Yeah, duh. Thanks for pointing that out.